styles/styles.css

```diff
@@ -1,13 +1,79 @@
-/* 
+/*
  * Put project specific base styles here.
  *
  * Note: The proect does not load this file.
  *       You will need to load these using scripts.js.
- * 
+ *
  *
  */
 :root {
+  --color-black: #000;
+  --color-gray-300: #D4D4D4;
   --color-quote: #367ea8;
+  --color-info-accent: #5C5CE0;
+  --color-info-accent-hover: #4646C6;
+  --color-info-accent-down: #3D3DB4;
+
+  /* body */
+  --body-color: var(--color-black);
+  --body-font-weight: 300;
+  --body-font-size-m: 1.125rem; /* 18px */
+  --body-font-size-s: 1rem; /* 16px */
+  --body-font-size-xs: 0.875rem; /* 14px */
+  --body-line-height: 1.5;
+
+  /* headings */
+  --heading-font-weight: 700;
+  --heading-line-height: 1.25;
+
+  /* grid sizes */
+  --grid-container-width: 83.4%;
+}
+
+main {
+  font-size: var(--body-font-size-m);
+  line-height: var(--body-line-height);
+  margin: 64px 0 0;
+  font-weight: var(--body-font-weight);
+}
+
+
+main h1, main h2, main h3, main h4, main h5, main h6 {
+  font-weight: var(--heading-font-weight);
+  line-height: var(--heading-line-height);
+  margin-top: 48px;
+  margin-bottom: 16px;
+}
+
+/* links */
+
+main a:any-link {
+  color: var(--body-color);
+  line-height: var(--body-line-height);
+}
```

Comment:
Same here. This makes general links blend into content which is an accessibility concern.